.travis.yml

```diff
@@ -77,6 +77,7 @@ env:
   - CLIENT=saucelabs:chrome COMMAND=test
   - CLIENT=saucelabs:safari:6 COMMAND=test
   - CLIENT="saucelabs:internet explorer:10:Windows 8" COMMAND=test
+  - CLIENT="saucelabs:MicrosoftEdge:20.10240:Windows 10" COMMAND=test
```

Comment:
No that's cool to change I think, if we put just `20` will it automatically update or do we need to use `latest`?